src/cmd-handlers.ts

```diff
@@ -70,12 +77,48 @@ function make_actions(reqFn: CallableFunction, body_args: Array<string> = [], in
     }
   }
 
-  function fill_obj(from: Record<string, any>, to: Record<string, any>, attributes: Array<string>) {
-    attributes.forEach(attr => {
-      to[attr] = from[attr] 
+  function modify_object(object: Record<string, any>, field: string, replace_for: string, from: Record<string, any> ) {
```

Comment:
this function does not seem necessary - why not just make the assignment directly inline?

Reply:
That's correct. I forgot to think with that simplicity after shrinking the function.